sources/packages/api/src/route-controllers/institution/institution.institutions.controller.ts

```diff
@@ -218,7 +219,7 @@ export class InstitutionInstitutionsController extends BaseController {
     }
     // Check if the user being added to th institution belongs to the institution.
     if (
-      institution.guid.toLowerCase() !==
+      institution.businessGuid.toLowerCase() !==
       bceidUserAccount.institution.guid.toLowerCase()
```

Comment:
Will trimming ensure no stone is left unturned ? @guru-aot and @ann-aot  any thoughts on this ?

Reply:
This information is not inputted by the user, it comes from the BCeID Web Service call and it is supposed to be sanitized.

Reply:
Got it.